doc/stdlib/math.rst

```diff
@@ -1684,6 +1684,12 @@ Statistics
 
    Compute the mean of whole array ``v``\ , or optionally along the dimensions in ``region``\ . Note: Julia does not ignore ``NaN`` values in the computation. For applications requiring the handling of missing data, the ``DataArray`` package is recommended.
 
+.. function:: mean(f::Function, v)
+
+   .. Docstring generated from Julia source
+
+   Apply the function `f` to each element of `v` and take the mean.
```

Comment:
should add one inline, otherwise "generated from Julia source" isn't true.

you can test make docs by tenporarily adding garbage to the rst version of the docstring here, and if it's working correctly then `make docs` will replace it with the inline docstring